db/version_edit.cc

```diff
@@ -227,6 +227,13 @@ bool VersionEdit::EncodeTo(std::string* dst) const {
       std::string unique_id_str = EncodeUniqueIdBytes(&unique_id);
       PutLengthPrefixedSlice(dst, Slice(unique_id_str));
     }
+    if (f.compensated_range_deletion_size) {
+      PutVarint32(dst, kCompensatedRangeDeletionSize);
```

Comment:
This change would add a new field `kCompensatedRangeDeletionSize` (how many bytes the range tombstones in the current file overlaps with older levels) and store it in`kNewFile4` records. Wondering if you have any concern/suggestion about this? @siying @pdillinger

Reply:
It looks OK to me.